plugins/index.md

```diff
@@ -273,26 +261,6 @@ deb http://deb.theforeman.org/ plugins {{page.version}}
 </pre>
 </div>
 
-<div class="installation_os installation_os_el7">
-  <p>
-    Change the version number in the URL to match the Foreman release in use.
-  </p>
-
-  <p>
-    To install a plugin:
-  </p>
-
-  <ol>
-    <li>Find the package for the plugin with the search function: <code>yum search discovery</code> or by checking the plugin documentation.</li>
-    <li>Install the package, e.g. <code>yum install tfm-rubygem-foreman_discovery</code>.</li>
-    <li>Restart Foreman with <code>touch ~foreman/tmp/restart.txt</code> or <code>systemctl restart foreman</code></li>
```

Comment:
Unrelated, but I don't think `touch ~foreman/tmp/restart.txt` works with Puma. Though perhaps it's better to focus on the new generation docs and make sure it's clean there.